edi_oca/models/edi_backend.py

```diff
@@ -284,6 +284,10 @@ def exchange_send(self, exchange_record):
         check = self._output_check_send(exchange_record)
         if not check:
             return "Nothing to do. Likely already sent."
+        # In case obsolete: skip sending
```

Comment:
I would do:
* remove this check completely
* modify the msg at line 286 as "Nothing to do. Likely already sent or obsolete." to leave the door opened w/ less impact
* as for the check see below the comment on `_check_obsolete`